examples/npm-workspaces/server/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
+    "esModuleInterop": true,
+    "strict": true,
+    "outDir": "dist",
+    "skipLibCheck": true,
+    "paths": {
+      "@/*": ["./src/*"]
+    }
```

Comment:
Try adding a baseDir here and in client. Using path aliases without a base is a bit sketchy imo

Reply:
Tried it, does not resolve the issue. I tried `baseUrl: "../"` as well (with adjusted path alias). Does not help either.

Also, https://www.typescriptlang.org/docs/handbook/modules/reference.html#baseurl

> baseUrl was designed for use with AMD module loaders. If you aren’t using an AMD module loader, you probably shouldn’t use baseUrl. Since TypeScript 4.1, baseUrl is no longer required to use [paths](https://www.typescriptlang.org/docs/handbook/modules/reference.html#paths) and should not be used just to set the directory paths values are resolved from.

So, it's actually recommended not so set baseUrl anymore.

Reply:
TIL about that, but it doesn't change the fact that I've experienced weird behavior without it 😭

I'll take a look when I get home and see if I can get it working

Reply:
Solved. See discord. I'll update the PR shortly.